listenbrainz/webserver/static/js/src/BrainzPlayer.tsx

```diff
@@ -409,8 +386,22 @@ export default class BrainzPlayer extends React.Component<
 
   trackInfoChange = (title: string, artist?: string): void => {
     this.setState({ currentTrackName: title, currentTrackArtist: artist });
+    const message = (
+      <>
+        <FontAwesomeIcon icon={faPlayCircle as IconProp} />
+        &emsp;{title}
+        {artist && ` — ${artist}`}
+      </>
+    );
+    this.handleInfoMessage(message);
```

Comment:
Show a notification with the song title on a blue background:
![image](https://user-images.githubusercontent.com/6179856/126175477-536c2d41-3385-431b-8827-71b6c2bbbe48.png)